### PR TITLE
Update dpimport_mriqc.sh

### PR DESCRIPTION
**Pull request by kcho**
The `csv` path on Line 37 needs to be inside quotation marks to avoid error.

### CODE REVIEW

**Comment by tashrifbillah** [comment]
It is correct in its current form. What error did you get?

**Comment by kcho** [comment]
> It is correct in its current form. What error did you get?

This is the error message that I get from running `dpimport_mriqc.sh` in the current main branch.

```
usage: import.py [-h] [-c CONFIG] [-d DBNAME] [-v] expr
import.py: error: unrecognized arguments: MRI_ROOT/derivatives/quick_qc/sub-YA01508/YA-YA01508-mriqc-day1to1.csv
```

My PR fixes this.

**Comment by tashrifbillah** [comment]
In which directory did you run it?

File: dpimport_mriqc.sh [hunk 1]
```diff
@@ -34,5 +34,5 @@ cd ${NDA_ROOT}
 import.py -c /data/predict/dpimport/examples/$CONFIG MRI_ROOT/derivatives/quick_qc/combined-*-mriqc-day1to*.csv
 
 # subject level data
-import.py -c /data/predict/dpimport/examples/$CONFIG MRI_ROOT/derivatives/quick_qc/**/??-???????-mriqc-day1to*.csv
+import.py -c /data/predict/dpimport/examples/$CONFIG "MRI_ROOT/derivatives/quick_qc/*/??-???????-mriqc-day1to*.csv"
```
Comment (tashrifbillah) [inline dpimport_mriqc.sh:37]: `quick_qc/*/` : are you confident that your CSV files will be at just one directory deep inside `quick_qc`?

File: dpimport_mriqc.sh [hunk 2]
```diff
@@ -34,5 +34,5 @@ cd ${NDA_ROOT}
 import.py -c /data/predict/dpimport/examples/$CONFIG MRI_ROOT/derivatives/quick_qc/combined-*-mriqc-day1to*.csv
```
Comment (tashrifbillah) [inline dpimport_mriqc.sh:34]: Please enclose `MRI_ROOT/derivatives/quick_qc/combined-*-mriqc-day1to*.csv` in `" "` as well.

**Comment by tashrifbillah** [comment]
Hi Kevin, thank you for contributing. It was actually my mistake. I have `" "` in other importers already.
https://github.com/AMP-SCZ/utility/blob/main/dpimport_eegqc.sh

**Review by tashrifbillah — CHANGES_REQUESTED** [review]
Tashrif has asked for a change.

File: dpimport_mriqc.sh (hunk 1 above)
Comment (kcho) [reply dpimport_mriqc.sh:37]: yes

**Review by kcho — COMMENTED** [review]
`"` added

**Review by kcho — COMMENTED** [review]
`"` added

**Comment by tashrifbillah** [comment]
Deployed.

**Pull request merged**